pytensor/tensor/math.py

```diff
@@ -277,6 +278,31 @@ def grad(self, inp, grads):
         return [x.zeros_like()]
 
 
+def argmax(x, axis=None, keepdims=False):
+    """
+    Returns indices of maximum elements obtained by iterating over given axis.
+
+    When axis is None (the default value), the argmax is performed
+    over the flattened tensor.
+
+    Parameters
+    ----------
+    keepdims : bool
+        If this is set to True, the axes which are reduced are left in
+        the result as dimensions with size one. With this option, the result
+        will broadcast correctly against the original tensor.
+
+    """
+    x = as_tensor_variable(x)
+    axis = CAReduce.normalize_axis(x, axis)
```

Comment:
calling the class method like this makes me think it should be a helper function outside `CAReduce`

Reply:
You're right